torchvision/datasets/kinetics.py

```diff
@@ -177,14 +177,15 @@ def _download_videos(self) -> None:
         if not check_integrity(split_url_filepath):
             download_url(split_url, file_list_path)
         list_video_urls = open(split_url_filepath)
+        list_video_urls = [str(line).replace(" ", "%20") for line in list_video_urls.readlines()]
```

Comment:
1. Use [`urllib.parse.quote`](https://docs.python.org/3/library/urllib.parse.html#urllib.parse.quote) instead of a manual solution
2. Maybe I'm missing something here, but since we overwrite `list_video_urls` we should be able to use a context manager here. Plus, instead of using `readlines` and manually stripping the newline delimiter away later, we can read the whole file and use `.split_lines()` to have it handled for us.

```suggestion
        with open(split_url_filepath) as file:
            list_video_urls = [urllib.parse.quote(line) for line in file.read().splitlines()]
```

Reply:
Okay, got it.